quicktests/overlaying/tests/basic/wheel.js

```diff
@@ -0,0 +1,46 @@
+function makeData() {
+  "use strict";
+
+  return [
+    { id: "0", r1: 0, r2: 1, t1: -55, t2: 55 },
+    { id: "1", r1: 0, r2: 1, t1: 180, t2: 300 },
+    { id: "2", r1: 0, r2: 1, t1: 70, t2: 150 },
+    { id: "3", r1: 1, r2: 2, t1: -40, t2: 20 },
+    { id: "4", r1: 1, r2: 2, t1: 250, t2: 280 },
+    { id: "5", r1: 1, r2: 2, t1: 80, t2: 160 },
+    { id: "6", r1: 2, r2: 3, t1: -60, t2: 60 },
+    { id: "7", r1: 2, r2: 3, t1: 180, t2: 220 },
+    { id: "8", r1: 3, r2: 4, t1: 60, t2: 180 }
+  ];
+}
+
+function run(svg, data, Plottable) {
+  "use strict";
+  var cs = new Plottable.Scales.Color();
+  var rScale = new Plottable.Scales.Linear().domain([0, 4]);
+  var tScale = new Plottable.Scales.Linear().domain([0, 360]);
+  var legend = new Plottable.Components.Legend(cs);
+
+  var wheel = new Plottable.Plots.Wheel()
+    .addDataset(new Plottable.Dataset(data))
+    .r(function(d){ return d.r1; }, rScale)
+    .r2(function(d){ return d.r2; }, rScale)
+    .t(function(d){ return d.t1; }, tScale)
+    .t2(function(d){ return d.t2; }, tScale)
```

Comment:
Try `t()` without the scale, since we're already in degrees. Also no scale is needed for `t2()`.